### PR TITLE
[PRN-259552: Added unit tests for API/Controllers]

### PR DESCRIPTION
**Pull request by shehzad-ismail8**

### CODE REVIEW

File: EPRN.UnitTests/API/Controllers/WasteControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            var journeyId = 5;
+            var monthSelected = 7;
+            _mockWasteService!.Setup(ws => ws.CreateJourney());
+
+            //Act
+            var result = await _wasteController!.SaveJourneyMonth(journeyId, monthSelected);
+
+            //Assert
+            Assert.IsNotNull(result);
+            Assert.IsInstanceOfType(result, typeof(OkResult));
```
Comment (NikPeters-Atos) [inline EPRN.UnitTests/API/Controllers/WasteControllerTests.cs:67]: There's a further test you can perform here: ensuring the waste service was called with the appropriate parameters

File: EPRN.UnitTests/API/Controllers/WasteControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        {
+            //Arrange
+            var journeyId = 3;
+
+            //Act
+            var result = await _wasteController!.WasteType(journeyId);
+
+            //Assert
+            Assert.IsNotNull(result);
+            Assert.IsInstanceOfType(result, typeof(OkObjectResult));
```
Comment (NikPeters-Atos) [inline EPRN.UnitTests/API/Controllers/WasteControllerTests.cs:32]: Test service was called

File: EPRN.UnitTests/API/Controllers/WasteControllerTests.cs [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            var monthSelected = 7;
+            _mockWasteService!.Setup(ws => ws.CreateJourney());
+
+            //Act
+            var result = await _wasteController!.SaveJourneyMonth(null, monthSelected);
+
+            //Assert
+            Assert.AreEqual(monthSelected, 7);
+            Assert.IsNotNull(result);
+            Assert.IsInstanceOfType(result, typeof(BadRequestObjectResult));
```
Comment (NikPeters-Atos) [inline EPRN.UnitTests/API/Controllers/WasteControllerTests.cs:87]: Test service was called

File: EPRN.UnitTests/API/Controllers/WasteControllerTests.cs (hunk 1 above)
Comment (shehzad-ismail8) [reply EPRN.UnitTests/API/Controllers/WasteControllerTests.cs:67]: Added

File: EPRN.UnitTests/API/Controllers/WasteControllerTests.cs (hunk 2 above)
Comment (shehzad-ismail8) [reply EPRN.UnitTests/API/Controllers/WasteControllerTests.cs:32]: Added

File: EPRN.UnitTests/API/Controllers/WasteControllerTests.cs (hunk 3 above)
Comment (shehzad-ismail8) [reply EPRN.UnitTests/API/Controllers/WasteControllerTests.cs:87]: Added

**Review by NikPeters-Atos — APPROVED** [review]
Noice

**Pull request merged**